x/concentrated-liquidity/lp_test.go

```diff
@@ -744,21 +743,16 @@ func (s *KeeperTestSuite) TestAddToPosition() {
 			amount1ToAdd: amount1PerfectRatio.QuoRaw(2),
 		},
 
-		// Error catching
-
-		"error: attempt to add to a position with underlying lock that has finished unlocking": {
+		"Add to a position with underlying lock that has finished unlocking": {
```

Comment:
What was the change that made this no longer error?

Reply:
So this was the test case that made me revise all the test cases actually. 
My first approach was that I noticed that this case should not error, where user should be able to add to a position with underlying lock that has finished unlocking. Then I fmt.Println along the lines for this test code, and noticed that this was not erroring due to the error type we have specified, but was erroring due to the account not having enough funds. 

Funding the account prior to calling add to positions fixed this